packages/dataviews/package.json

```diff
@@ -20,10 +20,12 @@
 	},
 	"main": "build/index.js",
 	"module": "build-module/index.js",
+	"calypso:src": "src/index.ts",
 	"exports": {
 		".": {
 			"types": "./build-types/index.d.ts",
 			"import": "./build-module/index.js",
+			"calypso:src": "./src/index.ts",
```

Comment:
The `calypso:src` condition should be the first one, before `types` and `import`. Order matters here, the resolver in webpack/TypeScript/Jest will use the first condition that matches. And we always want it to be the `calypso:src` one.

Reply:
pushed